src/frontend/components/report/ReportCard.vue

```diff
@@ -74,14 +68,24 @@ export default {
   },
   computed: {
     status () {
-      if (this.report.data.error) { return 'error' }
-      if (this.report.data.done) { return 'success' }
+      if (this.report.data.error) {
+        return 'error'
+      }
+      if (this.report.data.done) {
+        return 'success'
+      }
       return 'inprogress'
     },
     color () {
-      if (this.status === 'success') { return 'green darken-2' }
-      if (this.status === 'inprogress') { return 'blue darken-2' }
-      if (this.status === 'error') { return 'red darken-2' }
+      if (this.status === 'success') {
+        return 'green darken-2'
+      }
+      if (this.status === 'inprogress') {
+        return 'blue darken-2'
+      }
+      if (this.status === 'error') {
+        return 'red darken-2'
+      }
       return 'orange darken-2'
```

Comment:
You may want to use a switch

```suggestion
    color () {
      switch (this.status) {
        case 'success':
          return 'green darken-2';

        case 'inprogress':
          return 'blue darken-2';

        case 'error':
          return 'red darken-2';

        default:
          return 'orange darken-2';
      }
```